_components/button-group/guidance/variants/default/accessibility.md

```diff
@@ -0,0 +1,2 @@
+{:.usa-content-list}
```

Comment:
Is this class required? Not seeing it in previous Accordion or Banner.

Reply:
Nope, they're vestiges of some old work. I'll remove.

Reply:
Done. Removed others as well with a find/replace